Makefile

```diff
@@ -2,7 +2,8 @@
 test:
 	./check-fmt.sh
 	bazel build //...
-  	bazel test --test_output=errors //...
+	bazel test --test_output=errors //...
+	test/packages/test_complex_packages.sh
```

Comment:
looks like this needs to be "tests/packages".

You can probably leave this test out of the presubmits though, and leave it as a separate make target we run manually.